usaspending_api/etl/management/load_base.py

```diff
@@ -7,7 +7,7 @@
 from datetime import datetime
 from decimal import Decimal
 
-import dateutil
+from dateutil import parser
```

Comment:
After and investigation by kirk and I, it seems that this is the only way to get the store_value method to work with tests.